pkg/services/store/entity/README.md

```diff
@@ -0,0 +1,140 @@
+# Unified Storage
+
+The unified storage projects aims to provide a simple and extensible backend to unify the way we store different objects within the Grafana app platform.
+
+It provides generic storage for k8s objects, and can store data either within dedicated tables in the main Grafana database, or in separate storage.
+
+By default it runs in-process within Grafana, but it can also be run as a standalone GRPC service (`storage-server`).
+
+## Storage Overview
+
+There are 2 main tables, the `entity` table stores a "current" view of the objects, and the `entity_history` table stores a record of each revision of a given object.
+
+## Running Unified Storage
+
+### Baseline configuration
+
+The minimum config settings required are:
+
+```ini
+; dev mode is required
+app_mode = development
+; need to specify target here for override to work later
+target = all
+
+[server]
+; https is required for kubectl
+protocol = https
```

Comment:
is this really true?  it works for me without it.

Using `grafanaAPIServerEnsureKubectlAccess` it will setup your kubecfg for you... so maybe easier directions (until we have a nice helper in the UI)

Reply:
if you use `grafanaAPIServerEnsureKubectlAccess` and use the generate kubeconfig then you won't need https, but you also can't use grpc because that setup doesn't give you an id token.  I can look into what it would take to have an idtoken generated for requests that go via that route, but using a kubeconfig with an actual user associated also demonstrates that the user identity is passed through and comes back out in createdBy.